packages/AI/Vectors/Core/src/models/VectorBase.ts

```diff
@@ -46,21 +46,19 @@ export class VectorBase {
           throw new Error(`Entity with ID ${entityID} not found.`);
         }
     
-        const params = {
-          EntityName: entity.Name,
-          ResultType: 'entity_object' as const,
-          MaxRows: pageSize,
-          OffsetRows: Math.max(0, (pageNumber - 1) * pageSize),
-        };
-    
-        const rvResult: RunViewResult<T> = await this._runView.RunView<T>(params, this.CurrentUser);
+        const rvResult: RunViewResult<T> = await this._runView.RunView<T>({
+            EntityName: entity.Name,
+            ResultType: 'entity_object' as const,
+            MaxRows: pageSize,
+            OffsetRows: Math.max(0, (pageNumber - 1) * pageSize),
```

Comment:
@js-bc - this is fine for now, but for this PR change property to "StartRow" and in the future, the code in SQLServerDataProvider must be changed to use the sort order of the view as defined by the stored view and/or the RunViewParams. The proper way to do this is with a CTE that encapsulates the query and uses numbered rows in SQL Server (get ChatGPT to help) and then from there just picks the specific rows we want. You can DEFER doing the proper SQLServer approach for just now to get 2.0 out the door but we need to change this very soon after the 2.0 release